cds/modules/previewer/templates/cds_previewer/macros/player.html

```diff
@@ -14,7 +15,9 @@
         ui : {
           width : '100%',
           height: '100%',
-          fluid : true
+          {% if theo_config.fluid %}
```

Comment:
I think it is missing to have this `theo_config.fluid` true by default, and in embed.html set it to false, or am I missing something?

Reply:
Fluid is actually useless everywhere, we need to turn it off in the other pages as well because it might cut the video in smaller devices

Reply:
Okkkk, I would have removed it then...